src/repository/blip_repository.py

```
@@ -0,0 +1,16 @@
+from sqlalchemy.orm import Session
+from sqlalchemy import select
+from fastapi import Depends
+from src.infrastructure.database.connection import get_db, User
+
+class BLIPRepository:
+    def __init__(self, session : Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)